components/ws-manager-bridge/src/bridge.ts

```diff
@@ -474,12 +474,7 @@ export class WorkspaceManagerBridge implements Disposable {
 
             for (const [instanceId, ri] of runningInstancesIdx.entries()) {
                 const instance = ri.latestInstance;
-                if (
-                    !(
-                        instance.status.phase === "running" ||
-                        durationLongerThanSeconds(Date.parse(instance.creationTime), maxTimeToRunningPhaseSeconds)
```

Comment:
The `maxTimeToRunningPhaseSeconds` functionality is effectively covered by `contrlInstanceTimeouts` now, as it has specific timeouts for `preparing` and `building` phases.
The difference is that it no longer covers the other, ws-manager-goverened, pre-running phases (`pending`, `creating`, `initializing`). But as this was introduced a very long time ago as a bandaid to cover for a ws-manager bug, I think we're better of removing it.

Reply:
compare this diagram: https://www.notion.so/gitpod/WebApp-Architecture-784cf14bc76440e3beb8de56f61e7340#6c9bd05eef5a44408171a95b14ae73a9